src/main/java/org/breedinginsight/brapps/importer/services/processors/GermplasmProcessor.java

```diff
@@ -79,15 +79,15 @@ public class GermplasmProcessor implements Processor {
     List<BrAPIGermplasm> newGermplasmList;
 
     List<BrAPIGermplasm> updatedGermplasmList;
-    List<BrAPIGermplasm> existingGermplasms;
+    List<BrAPIGermplasm> existingGermplasm;
     List<List<BrAPIGermplasm>> postOrder = new ArrayList<>();
     BrAPIListNewRequest importList = new BrAPIListNewRequest();
 
-    public static String missingDbIdsMsg = "The following GIDs were not found in the database: %s.";
-    public static String missingParentalDbIdsMsg = "The following parental GIDs were not found in the database: %s.";
+    public static String missingGIDsMsg = "The following GIDs were not found in the database: %s.";
+    public static String missingParentalGIDsMsg = "The following parental GIDs were not found in the database: %s.";
     public static String missingParentalEntryNoMsg = "The following parental entry numbers were not found in the database: %s.";
     public static String badBreedMethodsMsg = "Invalid breeding method";
-    public static String missingEntryNumbersMsg = "Either all or none of the germplasm must have entry numbers.";
+    public static String missingEntryNumbersMsg = "Either all or none of the germplasm must have entry numbers";
```

Comment:
Was removing the period intended? Seems to be a mix in the messages, not sure if it would impact taf?

Reply:
It was intentional, I'll push a commit to remove the trailing period from the other messages as well. A period is added on the front end which results in a double period as you can see in the screenshots. I did a text search in TAF and I don't think these error messages are referenced anywhere.

![image](https://github.com/user-attachments/assets/fe777a59-f1a6-4b37-84c0-612867c3d73b)
![image](https://github.com/user-attachments/assets/adb61dda-2863-429f-abdb-6eacc8abef67)

Reply:
sounds good